CancerGov/_src/StyleSheets/_fonts.scss

```diff
@@ -64,6 +64,16 @@ $montserrat-font-stack: $montserrat, Arial, sans-serif;
 	url($fonts + "Montserrat-regular.svg") format("svg");
 }
 
+@font-face {
+	font-family: "DIN-Condensed-Regular";
+	font-weight: normal;
+	font-style: normal;
+	src: url($fonts + "DIN-Condensed-Regular.eot");
```

Comment:
The file names for the `src: url()` need to match the font file names which still have a lower case "r". Please change these back.

Reply:
The .eot files name also needs to be lowercase r